src/templates/ja/2019/contributors.html

```diff
@@ -2,7 +2,7 @@
 
 {% block title %}2019 貢献者 | HTTP ArchiveによるWeb Almanac{% endblock %}
 
-{% block description %}The {{ config.contributors.items() | length }} アナリスト、著者、ブレインストーマー、デザイナー、開発者、編集者、レビュアー、翻訳者として2019年のWeb Almanacに貢献した人々。{% endblock %}
+{% block description %} {{ config.contributors.items() | length }} アナリスト、著者、ブレインストーマー、デザイナー、開発者、編集者、レビュアー、翻訳者として2019年のWeb Almanacに貢献した人々。{% endblock %}
```

Comment:
Nit but can we remove the unnecessary space?

```suggestion
{% block description %}{{ config.contributors.items() | length }} アナリスト、著者、ブレインストーマー、デザイナー、開発者、編集者、レビュアー、翻訳者として2019年のWeb Almanacに貢献した人々。{% endblock %}
```